olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/VersionedLayerClient.h

```diff
@@ -192,6 +193,44 @@ class DATASERVICE_READ_API VersionedLayerClient final {
    */
   client::CancellableFuture<DataResponse> GetData(DataRequest data_request);
 
+  /**
+   * @brief Fetches data asynchronously using a TileKey.
+   *
+   * If the specified TileKey cannot be found in the layer, the callback is
+   * invoked with the empty `DataResponse` object (the `nullptr` result and an
+   * error). Version for request used from VersionedLayerClient constructor
+   * parameter. If no version is specified, the last available version is used
+   * instead.
+   * @note QuadTree with depth 4 are queried and stored in cache. It is
```

Comment:
I would write something like this:

This method in comparison to the GetData(DataRequest), will optimize the metadata query by requesting a QuadTree with depth 4 and store all subquads in cache. This way all further GetData() request that are contained within this QuadTree will profit from the already cached metadata. Please note that calling this method only makes sense if you have a persistent cache connected.